bundle-size/app.js

```diff
@@ -564,6 +564,60 @@ module.exports = app => {
     }
   });
 
+  v0.post('/commit/:headSha/report.json', async (request, response) => {
+    const {headSha} = request.params;
+    const {baseSha, bundleSizes} = request.body;
+
+    if (typeof baseSha !== 'string' || !/^[0-9a-f]{40}$/.test(baseSha)) {
+      return response
+        .status(400)
+        .end('POST request to /report must have commit SHA field "baseSha"');
+    }
+    if (
+      !bundleSizes ||
+      Object.values(bundleSizes).some(value => typeof value !== 'number') ||
+      !bundleSizes['dist/v0.js']
+    ) {
+      return response
+        .status(400)
+        .end(
+          'POST request to /report.json must have a key/value object ' +
+            'Map<string, number> field "bundleSizes", with at least one key ' +
+            'set to "dist/v0.js"'
+        );
+    }
+
+    // TODO(#271, danielrozenberg): this is a transitionary solution. This API
+    // endpoint accepts a JSON with multiple bundle sizes, but for now it only
+    // processes the size of dist/v0.js. Once we determine how we want to report
+    // and block PRs based on other bundle sizes we can start processing them.
+    const bundleSize = bundleSizes['dist/v0.js'];
+
+    const check = await getCheckFromDatabase(headSha);
+    if (!check) {
+      return response.status(404).end(`${headSha} not in database`);
```

Comment:
If this message appears in the status check or  Travis logs, perhaps expand on what database is being referred to?

Reply:
Done